content/master/operations/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Operations
+weight: 52
+state: alpha
+alphaVersion: v2.0-preview
```

Comment:
This is rendering with an extra `v` on https://deploy-preview-956--crossplane.netlify.app/master/operations/

> This feature was introduced in vv2.0-preview.

However, does it make more sense to use `alphaVersion: 2.0` instead of mentioning preview at all?